script.js

```diff
@@ -0,0 +1,57 @@
+let gridCount = 16;
+let grid = document.querySelector("#grid-container");
+const controls = document.querySelector("#controls");
+
+const resetBtn = document.createElement("button");
+resetBtn.classList.add("reset-button");
+resetBtn.textContent = "Reset";
+resetBtn.addEventListener("click", () => {
+  let cell = grid.lastElementChild;
+  while (cell) {
+    grid.removeChild(cell);
+    cell = grid.lastElementChild;
+  }
+  genDivs();
+});
+
+controls.appendChild(resetBtn);
+
+const gridCountBtn = document.querySelector("#resize-btn");
+gridCountBtn.textContent = "Resize ";
+gridCountBtn.addEventListener("click", userInput);
+
+function userInput() {
+  gridCount = Number(prompt("Enter size value"));
+  if (isNaN(gridCount) || gridCount < 16 || gridCount > 100) {
+    alert("Error: value must be a number between 16 and 100");
+    return;
+  }
+  let child = grid.lastElementChild;
+  while (child) {
+    grid.removeChild(child);
+    child = grid.lastElementChild;
+  }
+  genDivs();
+}
```

Comment:
Suggestion for the spacing
```suggestion
function userInput() {
  gridCount = Number(prompt("Enter size value"));
  
  if (isNaN(gridCount) || gridCount < 16 || gridCount > 100) {
    alert("Error: value must be a number between 16 and 100");
    return;
  }
  
  let child = grid.lastElementChild;
  
  while (child) {
    grid.removeChild(child);
    child = grid.lastElementChild;
  }
  
  genDivs();
}
```